Add buttons to open release pages when outdated

XanderXAJ opened the pull request:
Button opens the release's page on Github, e.g.:
https://github.com/atom/atom/releases/tag/v1.11.2

This then allows the user to download their file of choice.

![geiger-open-release-page](https://cloud.githubusercontent.com/assets/191109/19496204/6686bdcc-957e-11e6-9d9f-ee00cd4ef19d.png)

**Note:** I intended to also update the screenshot in the README, but my screen is such a small resolution that any screenshot I create will look jarring when compared to the other. ^^;

XanderXAJ committed:
Add buttons to open release pages when outdated

Button opens the release's page on Github, e.g.:
https://github.com/atom/atom/releases/tag/v1.11.2

This then allows the user to download their file of choice.

quasipedia reviewed (commented):
Wow XanderXAJ!  Thanks for this! :)  It was something I was meaning to do a long time, but I could not find documentation on the buttons and have had really shortage of time to code hobby projects...  I will try to test this out and do a release this coming weekend (I'm currently travelling). In the meantime, if you like, you could update the PR removing trailing semicolons (I can also do it, but I thought it was nicer if your name remained in git blame (praise!).  I can also re-do the screenshots myself if you have an hard time finding a Hi-res monitor, just tell me so and I will do. :)

XanderXAJ committed:
Remove semi-colons

XanderXAJ committed:
Remove downloads TODO from README

XanderXAJ committed:
Add contribution author

This feels remarkably difficult to do when it's myself...

XanderXAJ commented:
No worries quasipedia; I hope the changes work well for you. :)

I've pushed a few changes: I've removed the semicolons.  That said, am I missing something?  This is normally a more-or-less required part of JS, and I feel pretty bad for removing them...

I also removed the download TODO from the README. :D

Finally, if you're OK with it, I've added myself to the contributor list.  I don't require it, though.

About the screenshot: My highest resolution screen is 1080p, and there's no way I can normally run Atom at that size, so I'm afraid you might need to make the screenshot if you still want it to be a high resolution! 😅

quasipedia commented:
Hi @XanderXAJ, as you may have already noticed, I have now merged your changes and published v2.1.0.  Hope that works well for everybody, nice job! :)

About semicolons: that was mostly for uniformity and consistency with the codebase.  In ES6 I never came across a situation where they were needed to solve an ambiguity in code, so - having the choice to keep them or let them go - I went with the latter.  YMMV anyhow.  :)

I also tend to stick to lines that are below 80 chars, and always break those that exceed 100 (this is why I broke the ternary operator, in case you wonder).

Again: thanks a lot for your contribution! :)
